code/modules/mining/ores_coins.dm

```diff
@@ -460,7 +496,9 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 		to_chat(user, "<span class='notice'>You detach the string from the coin.</span>")
 	else ..()
 
-/obj/item/coin/welder_act(mob/user, obj/item/I)
+/obj/item/coin/wirecutter_act(mob/user, obj/item/I)
+	if(string_attached)
+		return
```

Comment:
Shouldn't this go under the `. = TRUE`?

Reply:
Can't get this to work properly, though im likely doing something wrong, feel free to poke me on discord about it.

Reply:
Could you make this do `return FALSE` then? That'll be the final nitpick here